.circleci/config.yml

```diff
@@ -495,18 +499,13 @@ workflows:
       - BUILD:
           filters:
             branches:
-              only: /^release\/.*/
-      - HOLD_FOR_APPROVAL:
-          type: approval
-          requires:
-            - BUILD
+              only: release
       - NPM_PUBLISH:
           requires:
-            - HOLD_FOR_APPROVAL
+            - BUILD
```

Comment:
why are you touching deploy release? @IbrahimCSAE

Reply:
this was by accident